src/pages/upload/Upload.css

```
@@ -4,6 +4,7 @@
     height: calc(100% - var(--top-bar-height));
     width: 100%;
     position: fixed;
+    padding-bottom: 16px;
```

this value is also used in Footer.css. Change it to a variable for height of footer and put it in index.css
```suggestion
    padding-bottom:var(--height-footer)
```